extension/storage/dbstorage/client.go

```diff
@@ -94,13 +98,16 @@ func (c *dbStorageClient) Batch(ctx context.Context, ops ...storage.Operation) e
 		case storage.Delete:
 			err = c.Delete(ctx, op.Key)
 		default:
+			_ = tx.Rollback()
 			return errors.New("wrong operation type")
```

Comment:
Let's keep one point for rollaback
```suggestion
			err = errors.New("wrong operation type")
```